packages/node-core/src/indexer/storeCache/csvStore.service.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020-2023 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: GPL-3.0
+
+import fs from 'fs';
+import path from 'path';
+import {stringify} from 'csv-stringify';
+
+export class CsvStoreService {
+  constructor(private modelName: string, private schema: string, private outputPath: string) {}
+
+  private getCsvFilePath(): string {
+    // TODO join relative to absolute
+    return path.join(this.outputPath, `${this.schema}-${this.modelName}.csv`);
```

Comment:
I think schema can be dropped from the name, the path should be responsible for this